sci-electronics/kicad/kicad-5.0.0.ebuild

```diff
@@ -0,0 +1,130 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+PYTHON_COMPAT=( python2_7 )
+
+WX_GTK_VER="3.0"
+
+inherit check-reqs cmake-utils eapi7-ver eutils flag-o-matic gnome2-utils python-single-r1 wxwidgets vcs-snapshot xdg
```

Comment:
also I do not see `flag-o-matic`  usage in your ebuild

Reply:
Correct, this is a left-over. Will remove.